wcfsetup/install/files/lib/system/worker/UserContentRemoveWorker.class.php

```diff
@@ -103,8 +114,6 @@ private function generateData() {
 			'count' => 0
 		];
 		
-		$contentProviders = ObjectTypeCache::getInstance()->getObjectTypes('com.woltlab.wcf.content.userContentProvider');
-		
 		// add the required object types for the select content provider
 		if (is_array($this->contentProvider)) {
```

Comment:
```suggestion
		if (is_array($this->contentProviders)) {
```
The variable should be renamed to be less confusing.